### PR TITLE
Mark appinsights as inactive

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

File: sdk/applicationinsights/azure-applicationinsights/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 0.1.1 (2022-02-11)
+
+* This package is no longer maintained. Please use https://pypi.org/project/azure-monitor-query/
```
Comment (scottaddie) [inline sdk/applicationinsights/azure-applicationinsights/CHANGELOG.md:5]:
```suggestion
* This package is no longer maintained. Use the [azure-monitor-query](https://pypi.org/project/azure-monitor-query/) package instead.
```

File: sdk/applicationinsights/azure-applicationinsights/README.md [hunk 2]
```diff
@@ -1,47 +1,5 @@
 ## Microsoft Azure SDK for Python
 
-This is the Microsoft Azure Application Insights Client Library.
+This package is no longer being maintained. Please use our latest package [azure-monitor-query](https://pypi.org/project/azure-monitor-query/).
```
Comment (scottaddie) [inline sdk/applicationinsights/azure-applicationinsights/README.md:3]:
```suggestion
This package is no longer being maintained. Use the [azure-monitor-query](https://pypi.org/project/azure-monitor-query/) package instead.
```

**Comment by rakshith91** [comment]
@scottaddie  Can you confirm we own this package? Or, if not, the owners are aware of this :)

File: sdk/applicationinsights/azure-applicationinsights/README.md [hunk 3]
```diff
@@ -1,47 +1,5 @@
...
-docs.microsoft.com.
-
-## Provide Feedback
-
-If you encounter any bugs or have suggestions, please file an issue in
-the [Issues](https://github.com/Azure/azure-sdk-for-python/issues)
-section of the project.
-
-![image](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2Fazure-applicationinsights%2FREADME.png)
+A [migration guide](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/monitor/azure-monitor-query/migration_guide_app_insights.md) has been provided for the same.
```
Comment (scottaddie) [inline sdk/applicationinsights/azure-applicationinsights/README.md:5]:
```suggestion
For migration instructions, see the [migration guide](https://aka.ms/azsdk/python/migrate/ai-to-monitor-query).
```

**Comment by scottaddie** [comment]
@rakshith91 Yes, the service PM is aware that we're deprecating this package.

**Pull request merged**